frontend/src/components/permissions/components/edit-permission-element/edit-permission-element.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { useForm } from 'react-hook-form';
+import { Checkbox } from 'components/common';
+import { IPermission, IUserWithPermissions } from 'common/interfaces';
+import { PermissionsActionCreator } from 'store/slices';
+import { PermissionName } from 'common/enums';
+import { RootState } from 'common/types';
+import { getPermissionsNames } from 'components/permissions/helpers';
+
+type Props = {
+  user: IUserWithPermissions;
+  nameOfPermission: string;
+};
+
+const EditPermissionElement: React.FC<Props> = ({ user, nameOfPermission }) => {
+  const { permissions } = useSelector(({ permissions }: RootState) => ({
+    permissions: permissions.permissions,
+  }));
+  const dispatch = useDispatch();
+
+  const { errors, register, getValues } = useForm<Record<string, string | boolean>>({
+    defaultValues: {
+      [nameOfPermission]: getPermissionsNames(user).includes(nameOfPermission as PermissionName),
+    },
+    mode: 'onChange',
+  });
+
+  const handleChange = () => {
+    const permission: IPermission | undefined = permissions.find((permission) => permission.name === nameOfPermission);
```

Comment:
```suggestion
    const permission = permissions.find((permission) => permission.name === nameOfPermission);
```

Reply:
Moe it to the `frontend/src/components/permissions/components/edit-permission-element/helpers`
`getPermissionByName`